src/app/tests/TestCommandInteraction.cpp

```diff
@@ -1067,9 +1100,15 @@ void TestCommandInteraction::TestCommandHandlerInvalidMessageAsync(nlTestSuite *
     err = commandSender.SendCommandRequest(ctx.GetSessionBobToAlice());
     NL_TEST_ASSERT(apSuite, err == CHIP_NO_ERROR);
 
+    // Warning! This test is not testing what the testname is suggesting. Because the invoke request is malformed,
```

Comment:
Why not just rename the test?

Reply:
Because it then just becomes `TestCommandHandlerInvalidMessageSync` which is the test above. I have a plan to fix this test in my next PR with the sender